### DISCUSSION TITLE
Thermal Goggles Break the Atmosphere/Fun-Factor

### DISCUSSION THREAD

**Discussion by aeldred28** [FakeFishGames/Barotrauma#6815] [Unstable feedback]
There hasn’t been much discussion surrounding the balancing of items introduced with this update. I’d like to start a thread on what I think is an extremely broken item that ruins multiplayer and singleplayer gameplay; the thermal goggles…

Thermal goggles being able to completely negate LOS on servers that have it enabled feels very much like cheating — like turning on debug draw while clearing a wreck.

The best part of Wrecks, Ruins, and Abandoned Outposts is cutting through doors and walls, while being afraid of what is waiting on the other side. It used to be the case that players only had audio cues to warn them of what is on the other side of a wall. And that tension is essentially the entire fun-factor behind these missions. 

(You can see a very good example of this in the last Twitch stream Criken did)

But now players have an extremely easy to acquire item that kills the tension and makes these missions a chore. Players know exactly where every single enemy on the wreck/ruin/outpost is, even ones 3 floors below them…

Another subtle thing to consider is multiplayer, particularly traitor missions:

If captains and crew can effectively see everyone at all times like they were playing with LOS turned off, then there is simply no way to hide or be sneaky while completing a traitor objective. Everyone will know when the traitor is hanging out in the ballast growing raptor eggs or etc… 

 The “thermal goggles” don’t actually function like thermal goggles — more like psychic goggles, or x-ray. 

Players always had the option of playing with LOS off, but this item does more harm than good on most servers and campaigns that prefer LOS because of how it affects atmosphere. 

There has been a few solutions proposed:

-Make the goggles apply LOS. The item would still be very useful as it would follow the progression of the flashlight. allowing you to see enemies in the complete darkness outside the sub with ease, just like real thermal vision would. 

-Make the goggles require batteries that run out approximately as fast as oxygen tanks. This would force players to only use the item when they know they need it. However the tension of wrecks and ruins would still be lost because the players will always use them to spot all the enemies in a location before they enter. 

-Make the goggles use the old diving-mask vision ‘mask’.
 This would stop the goggles from being 360° and stop players from using them at all times. However the goggles would still be overpowered in wrecks and ruins for the same reasons above. 

-Make the goggles very short range. This would stop them from being so OP that players can see an entire outpost/wreck from 3 floors away. 

I hope the team considers some of these balancing ideas for the update. personally the goggles have been the strongest feedback I have so far because of my experience using them on wrecks/ruins/outposts.

**Comment by Udrakan** [comment]
I like this one in particular: -Make the goggles use the old diving-mask vision ‘mask’.
This would be a nice and unique effect for the goggles.
Also, another problem with thermals is players seeing monsters spawn through walls when approaching some wrecks when they reach the treshold.

**Comment by Cosmos-11** [comment]
IMO I don't like the Diving mask vision but I would like to add, make it so if you are in the water and there are no rocks around you, you can see 360° but if you are in a ship theres a certain ammount of things you see through like you can see through 1 floor but not 2. And ship wrecks and ruins, make it so you cant see trough them or make it so you can see very little and if you have  Psychosis you see some fake signals behind walls/doors.

**Comment by trevormoney1** [comment]
I can definitely see the thermal goggles as they are being overpowered and think it should be reworked. However, I like the really powerful effect of seeing enemies and players through walls. Perhaps the current thermal goggle effect could be reworked into Watcher Genetic Material? Please give feedback on this, I might make it a suggestion or someone else can. (Could also make watcher genetic material an R ability similar to mudraptor and spineling DNA.

**Reply by aeldred28** [reply]
That would be helpful as an obstacle towards getting the effect: Players would have to actually find the genetic material in order to get the upgrade, rather than being able to buy it from the start of a campaign for cheap. 

I see that the devs changed the effect of the goggles this latest build, but I still think it should be nerfed further; either by making the goggles hard to get, or doing something like in your suggestion.

**Comment by StruggleDuck** [comment]
IMO, I agree with some of the proposed changes, but not all. I feel like making the goggles really short range would just make them really clunky to use, combined with giving them a small fov. Instead, I feel that they should leave the range as it is, but make it so that certain creatures have lower heat signatures than others, as well as making heat signatures more visible when they are out of water, and less visible when in water. The other thing I think they should keep is using them through walls. Without that, the thermal goggles essentially become useless, as that is their primary purpose. Without that, all it would do is limit the players vision and take away potential protection/buffs that other clothing items could provide.

**Reply by aeldred28** [reply]
If the goggles didn’t see through walls they wouldn’t be useless. They would have the same benefit/function that real thermal optics do — in that they make targets much easier to see when not obstructed, thus removing the need for a flashlight. 

Being able to see through walls breaks way too much to be worth it. Turning off LOS is already an option.
